dcos/rpcclient.py

```diff
@@ -145,7 +144,8 @@ def http_req(self, method_fn, path, *args, **kwargs):
                 request_method=e.response.request.method,
                 request_url=e.response.request.url,
                 json_body=json_body)
-            raise DCOSException(message)
+
+            raise DCOSException(message) from e
```

Comment:
I can't speak for shakedown. But the internals of the CLI (ie stacktrace) are not the right abstraction layer for your debugging. What information do you want from the CLI that you are missing?